### PR TITLE
Fixed wrong totals WA

### PR DESCRIPTION
**Pull request by aed3**

### CODE REVIEW

**Comment by praging** [comment]
closes https://github.com/covidatlas/coronadatascraper/issues/557

**Comment by praging** [comment]
closes https://github.com/covidatlas/coronadatascraper/issues/557

File: src/shared/scrapers/USA/WA/index.js [hunk 1]
```diff
@@ -180,9 +180,12 @@ const scraper = {
...
 
-      counties.push(transform.sumData(counties));
-      counties = geography.addEmptyRegions(counties, this._counties, 'county');
-      return counties;
+      const totals = {
+        cases: data.features[0].attributes.CV_State_Cases,
+        deaths: data.features[0].attributes.CV_State_Deaths
+      };
+      counties.push(totals);
+      return geography.addEmptyRegions(counties, this._counties, 'county');;
```
Comment (praging) [inline src/shared/scrapers/USA/WA/index.js:188]:
```suggestion
      return geography.addEmptyRegions(counties, this._counties, 'county');
```

File: src/shared/scrapers/USA/WA/index.js [hunk 2]
```diff
@@ -156,8 +156,8 @@ const scraper = {
       counties.push(transform.sumData(counties));
       return counties;
     },
-    '2020-03-30': async function() {
-      let counties = [];
+    '2020-3-30': async function() {
```
Comment (lazd) [inline src/shared/scrapers/USA/WA/index.js:159]:
```suggestion
    '2020-03-30': async function() {
```

**Pull request merged**